fms2_io/include/get_variable_attribute.inc

```diff
@@ -64,7 +74,11 @@ subroutine get_variable_attribute_0d(fileobj, variable_name, attribute_name, &
   select type(attribute_value)
     type is (character(len=*))
       call mpp_broadcast(charbuf, len(charbuf), fileobj%io_root, pelist=fileobj%pelist)
-      call string_copy(attribute_value, charbuf(1), check_for_null=.true.)
+      if (reproduce_null_char_bug) then
+         call string_copy(attribute_value, charbuf(1), check_for_null=.false.)
+      else
+         call string_copy(attribute_value, charbuf(1), check_for_null=.true.)
+      endif
```

Comment:
Is this for all attributes?  I thought the only ones that mattered were `lat_bnds` and `lon_bnds` for this bug.

Reply:
`reproduce_null_char_bug` is only going to be true if it is passed in the `get_variable_attribute`. 

This only happens in axis_utils:
https://github.com/NOAA-GFDL/FMS/blob/fa87962ce034cb2cf985e9cfe19d8983ba97273f/axis_utils/axis_utils2.F90#L180
https://github.com/NOAA-GFDL/FMS/blob/fa87962ce034cb2cf985e9cfe19d8983ba97273f/axis_utils/axis_utils2.F90#L190

So it is not for all attributes...